app/assets/stylesheets/helpers/_reset.scss

```diff
@@ -1,29 +1,115 @@
 // flatten browser default padding and font-sizing so we can cleanly apply our own
 
-/* Borrowed from http://meyerweb.com/eric/tools/css/reset/ */
-div, span,
-h1, h2, h3, h4, h5, h6, p, blockquote, pre,
-a, abbr, acronym, address, big, cite, code,
-del, dfn, em, img, ins, kbd, q, s, samp,
-small, strike, strong, sub, sup, tt, var,
-b, u, i, center,
-dl, dt, dd, ol, ul, li,
-fieldset, form, label, legend,
-table, caption, tbody, tfoot, thead, tr, th, td,
-article, aside, canvas, details, embed,
-figure, figcaption, footer, header, hgroup,
-menu, nav, output, ruby, section, summary,
-time, mark {
+// Borrowed from http://meyerweb.com/eric/tools/css/reset/
```

Comment:
@fofr I've removed the `_reset.scss` file and I can see differences in some pages (resolution used: 1280x1500). Here is an example for https://www.gov.uk/browse

Live:

![1280_phantomjs_live](https://cloud.githubusercontent.com/assets/416701/21226876/6d2cdb92-c2cf-11e6-9bf9-31a2c3c6ed18.png)

Dev (without `_reset.scss`):

![1280_phantomjs_dev](https://cloud.githubusercontent.com/assets/416701/21226882/720819b0-c2cf-11e6-8e14-d6f986b53461.png)